.circleci/.docirclescript.sh

```diff
@@ -0,0 +1,261 @@
+#!/bin/sh
+
+# The temporary directory where we want to clone to is the first argument
+# to this script
+if [ "$#" -lt "1" ]
+then
+	echo "FATAL: Temporary directory not specified!" 1>&2
+	exit 100
+fi
+__tempdir="$1"
+
+# The repository lives here
+__baserepo="$(pwd)"
+
+# The entire release will be mocked in another directory
```

Comment:
The script creates commits and the Maven release plugin also creates commits and tags. I create a new repository so that the repository that CircleCI is not messed with when the release process is tested for each build.

Reply:
Can Maven release on an existing commit/tag?

Reply:
Maven releases on a branch or tag. The preparation step creates the tag and modifies the POM accordingly, then the actual release process finds that tag from the remote and builds POM/JAR from that to upload to Bintray.

Reply:
Re: Git clone copy. Wouldn't breaking these steps up into separate jobs solve the issue where you have git repos in different states? The `checkout` step isn't shared between jobs and it is a forced pull.